### PR TITLE
docs: SQL tools

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

as title

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/reference/sql-tools.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+You can use the `psql` command line tool to connect to the GreptimeDB.
+Please refer to the [PostgreSQL protocol](/user-guide/protocols/postgresql.md) document for connection information.
+
+After you connect to the server, you can use all [GreptimeDB SQL commands](/reference/sql/overview.md) to interact with the database.
+
+## GreptimeDB Dashboard
+
+You can run SQL and visualize data in the [GreptimeDB Dashboard](/getting-started/installation/greptimedb-dashboard.md).
+
+## GUI tools
```
Comment (sunng87) [inline docs/reference/sql-tools.md:274]: superset/metabase are not *sql tools*, we should use dbeaver here

File: docs/reference/sql-tools.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
+# SQL Tools
+
+GreptimeDB uses SQL as its main query language and supports many popular SQL tools.
+This document guides you on how to use SQL tools with GreptimeDB.
+
+## Programming languages
```
Comment (sunng87) [inline docs/reference/sql-tools.md:6]:
```suggestion
## Language drivers
```

**Review by fengjiachun — APPROVED** [review]
LGTM

File: docs/reference/sql-tools.md (hunk 1 above)
Comment (nicecui) [reply docs/reference/sql-tools.md:274]: got it, I will remove the GUI tools section and add dbeaver in the future.

**Comment by nicecui** [comment]
@sunng87 PTAL

**Review by sunng87 — APPROVED** [review]
We can add dbeaver in #1192

**Pull request merged**